x-pack/plugins/security_solution/server/lib/telemetry/tasks/security_lists.ts

```diff
@@ -71,7 +76,14 @@ export function createTelemetrySecurityListTaskConfig(maxTelemetryBatch: number)
             licenseInfo,
             LIST_TRUSTED_APPLICATION
           );
-          count += trustedAppsJson.length;
+          tlog(logger, `Trusted Apps: ${trustedAppsJson}`);
```

Comment:
I think it's better not to log the whole JSON but just the relevant info, eg num or records, WDYT?